blivet/threads.py

```diff
@@ -40,7 +41,7 @@ def run_with_lock(*args, **kwargs):
                 exn_info = get_thread_exception()
                 if exn_info[1]:
```

Comment:
py2 it's OK to do `some_exception[N]`, but in py3 you need to do `some_exception.args[N]`, which also works for py2 as well.